src/store/modules/inbox.js

```diff
@@ -28,6 +25,11 @@ const getters = {
         dayRead = new Date(lastMessage.receivedTime).toLocaleDateString("En", {
           weekday: "short",
         });
+      } else if (item.lastRead !== null) {
+        //to account for empty conversation/inbox item, so it doesn't remain always unread
+        dayRead = new Date(item.lastRead).toLocaleDateString("En", {
```

Comment:
Instead of instantiating the Dates twice (one in the if statement and the other in the if/else clauses), would there be a way to declare it once?

Lines 23, 25 and 30

Reply:
Good catch, I'll be right on it!